src/Equinox.EventStore/EventStore.fs

```diff
@@ -623,7 +623,8 @@ type ConnectionStrategy =
 type Connector
     (   username, password, reqTimeout: TimeSpan, reqRetries: int,
         [<O; D(null)>]?log : Logger, [<O; D(null)>]?heartbeatTimeout: TimeSpan, [<O; D(null)>]?concurrentOperationsLimit,
-        [<O; D(null)>]?readRetryPolicy, [<O; D(null)>]?writeRetryPolicy,
+        [<O; D(null)>]?readRetryPolicy, [<O; D(null)>]?writeRetryPolicy, [<O; D(null)>]?clientConnectionTimeout,
```

Comment:
can you add a line break after writeRetryPolicy please?
prob should be on same line as gossipTimeout ?
as the gossip happens before the connect maybe also swap the order (ditto order of application below)

Reply:
I don't think that order in that case really matter, but I'll change it

Reply:
on reflection, given gossip timeout is only relevant for gossip, the order of the two args probably makes sense as it is
(also arguably the gossip options could or should go into the Discovery union, but thats probably a bridge too far given the next step is to mint a `Equinox.EventStore.Grpc` for ES v6 ;) )

Reply:
Changed

Reply:
Thanks, will merge (unless you want to write the changelog.md entry first...)

Reply:
Updated `CHANGELOG.md`

Reply:
Wonderful, thanks!, merging shortly